src/elementary/ReactionID.hpp

```diff
@@ -48,6 +48,14 @@ namespace elementary {
     /* methods */
     #include "elementary/ReactionID/src/validate.hpp"
 
+    /**
+     *  @brief Return whether or not the identifier is a special reaction

```

Comment:
What is a "special" reaction?

Reply:
Reactions for which we cannot generate a particle based identifier: things like total, mt5, fission, etc.